Makefile

```diff
@@ -54,3 +54,21 @@ devnet-down:
 
 fmt-contracts:
 	cd suave && forge fmt
+
+release:
+	# Generate the changelog for the release
+	./suave/scripts/parse-changelog.sh $(TAG) > /tmp/changelog.md
+
+	# Build and publish the release
+	docker run \
+		--rm \
+		-e CGO_ENABLED=1 \
+		-e GITHUB_TOKEN="$(GITHUB_TOKEN)" \
+		-v /tmp/changelog.md:/build/changelog.md \
+		-v /var/run/docker.sock:/var/run/docker.sock \
+		-v $(HOME)/.docker/config.json:/root/.docker/config.json \
+		-v `pwd`:/go/src/$(PACKAGE_NAME) \
+		-v `pwd`/sysroot:/sysroot \
+		-w /go/src/$(PACKAGE_NAME) \
+		ghcr.io/goreleaser/goreleaser-cross:v1.19.5 \
+		release --clean --release-notes /build/changelog.md
```

Comment:
i think it'd be better to create a draft release here, and manually edit the description and publish when ready